src/components/VideoThumbnail/VideoThumbnail.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import {useI18n} from '../../utilities/i18n';
+import {
+  secondsToTimeComponents,
+  secondsToTimestamp,
+  secondsToDurationTranslationKey,
+} from '../../utilities/duration';
+
+import {PlayIcon} from './illustrations';
+import styles from './VideoThumbnail.scss';
+
+export interface VideoThumbnailProps {
+  /** URL source for thumbnail image. */
+  thumbnailUrl: string;
+  /** Length of video in seconds. */
+  videoLength?: number;
+  /** Custom ARIA label for play button.
+   * @default 'Play video of length {human readable duration}'
+   */
+  accessibilityLabel?: string;
+  /** Callback on click or keypress of thumbnail. Use to trigger render of the video player in your chosen format, for example within a modal or fullscreen container. */
+  onClick(): void;
+  /** Callback on mouse enter, focus, or touch start of thumbnail. Use to trigger video preload. */
+  onBeforeStartPlaying?(): void;
+}
+
+export const VideoThumbnail = ({
```

Comment:
We use function declarations (`export function VideoThumbnail() {}` instead of function expressions `export const VideoThumbnail = () => {}` every where else, can you please stick to that convention